lib/rust/ensogl/component/grid-view/src/entry/visible.rs

```diff
@@ -78,11 +78,11 @@ where EntryParams: frp::node::Data
     pub fn create_entry<E: Entry<Params = EntryParams>>(
         &self,
         text_layer: Option<&Layer>,
-    ) -> VisibleEntry<E> {
+    ) -> (VisibleEntry<E>, Option<frp::Source<()>>) {
```

Comment:
The docs do not explain what is the second value returned. Also, there should be an explanation why we're doing it this way.